### PR TITLE
Update logging config

### PR DESCRIPTION
**Pull request by staskolauren**
Stop using `skripts` and `@therockstorm/utils` as they have not been updated since Feb 2021 / Oct 2020 respectively. The are pulling in outdated subdependencies used for dev tools within this repo (e.g. the [0.0.22 version of `skripts`](https://github.com/therockstorm/skripts/blob/7f54735f42bbabf15452f76364842c79b96a32b6/package.json#L3) that [this repo recently updated to](https://github.com/Dwolla/cloudwatch-alarm-to-slack/commit/4072e1be51e54c762aa876b42e2264c4b23273bd) uses `ts-lint` which has been deprecated in favor of `eslint`). Instead manage dev tool dependencies and their configuration in this package.

* remove `@therockstorm/utils`, use traditional `console.log` and `console.error` instead
* remove `skripts` - configure eslint, jest, tsconfig and webpack from this repository instead

### CODE REVIEW

File: src/fetcher.ts [hunk 1]
```diff
@@ -26,19 +23,20 @@ export const fetch = async (a: Alarm): Promise<string> => {
   const r = await asyncRetry<GetQueryResultsResponse>(
     async () => {
       const res = await cwl.getQueryResults({ queryId: id as string }).promise()
-      return res.status === "Running" ? thrw(new Error(res.status)) : res
+      return res.status === "Running"
+        ? console.error(new Error(res.status))
```
Comment (ljredmond-dwolla) [inline src/fetcher.ts:27]: I think the `thrw` method that's being replaced would actually throw an Error here

Comment (gorskilauren) [reply src/fetcher.ts]: Good call! Updated!

**Comment by gorskilauren** [comment]
If anyone's curious - [this](https://github.com/Dwolla/cloudwatch-alarm-to-slack/compare/just-remove-utils?expand=1) is what it would look like to just migrate away from @therockstorm/utils and use the latest version of skripts.
Btw, if we stop using @therockstorm/utils , our npm run test check [fails](https://ci.dwolla.net/blue/organizations/jenkins/GitHub%2Fcloudwatch-alarm-to-slack/detail/update-logging-config/1/pipeline) on the skripts tslint part,   specifically from Calls to 'console.error' are not allowed. . I struggled trying to turn off that rule (I couldn't find a way to cleanly override the default tslint config in our current version of `skripts`, and I think [this file](https://github.com/Dwolla/cloudwatch-alarm-to-slack/blob/main/tslint.yml) is being ignored). I saw `ts-lint` has been deprecated in favor of `eslint` , which is how I ended up trying to upgrade and/or move away from skripts.

**Comment by ljredmond-dwolla** [comment]
Running `npm test`, I noticed that the `SLACK_WEBHOOK_URL` environment variable is not being resolved in the CloudFormation template that's generated in the `.serverless` folder. I think this bit of `serverless.js`:

```javascript
SLACK_WEBHOOK_URL: "${ env: SLACK_WEBHOOK_URL }"
```

should be:

```javascript
SLACK_WEBHOOK_URL: "${env:SLACK_WEBHOOK_URL}"
```

I think this probably started when the Serverless Framework version was updated. It's not directly related to the changes in this PR, so if we'd rather keep it separate I can get another PR up to address this.

**Comment by gorskilauren** [comment]
> Running `npm test`, I noticed that the `SLACK_WEBHOOK_URL` environment variable is not being resolved in the CloudFormation template that's generated in the `.serverless` folder. I think this bit of `serverless.js`:
> 
> ```js
> SLACK_WEBHOOK_URL: "${ env: SLACK_WEBHOOK_URL }"
> ```
> 
> should be:
> 
> ```js
> SLACK_WEBHOOK_URL: "${env:SLACK_WEBHOOK_URL}"
> ```
> 
> I think this probably started when the Serverless Framework version was updated. It's not directly related to the changes in this PR, so if we'd rather keep it separate I can get another PR up to address this.

Good call on this as well! 🎉  I'd say the goal of this PR is to get the cloudwatch-alarm-to-slack lambda in a working state again, so I went ahead and updated now!

**Review by aquilesg — APPROVED** [review]

**Pull request merged**